gson/src/main/java/com/google/gson/internal/bind/DateTimeTypeAdapters.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright (C) 2011-2016 Google Inc.
```

Comment:
Ok, I am fine with 2011 (though I think the first public release of Gson was in 2009).